### ISSUE TITLE
Can't update from 3.3.1 to 4.0.4 via NuGet (Xamarin.Android)

### ISSUE THREAD

**Issue by tbalcom** [AutoMapper/AutoMapper#916]
I have a Xamarin project using Xamarin.Android 5.1.6.7, Xamarin 3.11.1443.0, and Mvvm Cross 3.5.1.  I have the typical MvvmCross solution setup: a Core PCL project, a 'Droid' Android project, and a Core.Tests Unit Test project.  I am able to update AutoMapper via NuGet for the PCL Core project without any issues.  I am not able to update AutoMapper via NuGet for the Droid project.  I get the following error:

```
PM> Update-Package -Id Automapper -ProjectName Dispatch.Droid
Updating 'AutoMapper' from version '3.3.1' to '4.0.4' in project 'Dispatch.Droid'.
Removing 'AutoMapper 3.3.1' from Dispatch.Droid.
Successfully removed 'AutoMapper 3.3.1' from Dispatch.Droid.
Adding 'AutoMapper 4.0.4' to Dispatch.Droid.
Update-Package : Failed to add reference to 'System.Runtime'. Please make sure that it is in the Global Assembly Cache.
At line:1 char:15
+ Update-Package <<<<  -Id Automapper -ProjectName Dispatch.Droid
    + CategoryInfo          : NotSpecified: (:) [Update-Package], InvalidOperationException
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PowerShell.Commands.UpdatePackageCommand
```

**Comment by tbalcom** [comment]
While troubleshooting this I decided to remove AutoMapper via NuGet and start over.  The packages.config file changed when I did this.  The change was

From:
`<package id="AutoMapper" version="3.3.1" targetFramework="portable-net45+sl50+MonoAndroid10+xamarinios10+MonoTouch10" />`
To:
`<package id="AutoMapper" version="3.3.1" targetFramework="portable-net45+win+MonoAndroid10+xamarinios10+MonoTouch10" /`
In other words, 'sl50' was replaced with 'win'.  Now when I try to add 4.0.4 via NuGet I get the same error except it's now referencing System.Linq instead of System.Runtime.

**Comment by jbogard** [comment]
This is a bad nuspec on my part, fixed in 4.1.1.
